packages/sitecore-jss-react/src/components/FEaaSComponent.tsx

```diff
@@ -73,25 +77,27 @@ export const FEaaSComponent = (props: FEaaSComponentProps): JSX.Element => {
   }
 
   let data: { [key: string]: unknown } = null;
-  if (props.params?.ComponentDataOverride) {
-    // Use override data if provided
-    try {
-      data = JSON.parse(props.params.ComponentDataOverride);
-    } catch (e) {
-      data = null;
+  if (props.fetchedData === null || props.fetchedData === undefined) {
```

Comment:
You are checking explicitly for _null_, _undefined_. Are you doing that because there can be an empty string value?

Reply:
When there is an error during fetch, `fetchedData` would be null.
Checking undefined for the way our test are setup. In unit test `fetchedData` comes out to be undefined when we are not doing SSR.